src/main/java/frc/robot/commands/ShootNoteCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ScoringConstants;
+import frc.robot.subsystems.Scoring;
+
+public class ShootNoteCommand extends Command {
+    private Scoring scoring = Scoring.getInstance();
+    private Timer timer = new Timer();
+    private int secondsToScore;
+
+    public ShootNoteCommand(int secondsToScore) {
+        this.addRequirements(scoring);
```

Comment:
The "this" keyword is not required when you are adding requirements.

Reply:
I agree its not required but its not necessarily bad to include it. I would have a conversation with the whole subteam about this (pun intended)
One argument is it's not necessary so don't include it. Another could be that including it anyways makes it more obvious where that function is coming from to a beginner